ci_tutorial/packages_pipeline/single_configuration.rst

```diff
@@ -0,0 +1,98 @@
+Package pipeline: single configuration
+======================================
+
+We will start with the most simple case, in which we only had to build 1 configuration, and that configuration 
+can be built in the current CI machine.
+
+As we described before while presenting the different server binary repositories, the idea is that package builds
+will use by default the ``develop`` repo only, which is considered the stable one for developer and CI jobs.
+
+Let's make sure we start from a clean state:
+
+.. code-block:: bash
+
+    $ conan remove "*" -c  # Make sure no packages from last run
+    $ conan remote remove "*"  # Make sure no other remotes defined
+    $ conan remote add develop <url-develop-repo>  # Add only the develop repo
+
+
+The removal and addition of repos accross this tutorial can be a bit tedious, but it is important for the correct
+behavior. Also, there might be other configurations that can be even more efficient for some cases, like re-triggering
+a broken job because of CI malfunction, but we will keep it simple at the moment and try to focus on the main concepts.
+
+With this configuration the CI job could just do:
+
+.. code-block:: bash
+
+    $ conan create ai --build="missing:ai/*"
+    ...
+    ai/1.1.0: SUPER BETTER Artificial Intelligence for aliens (Release)!
+    ai/1.1.0: Intelligence level=50
+
+
+Note the ``--build="missing:ai/*"`` might not be fully necessary in some cases, but it can save time in other situations.
+For example, if the developer did some changes just to the repo README, and didn't bump the version at all, Conan will not 
+generate a new ``recipe revision``, and detect this as a no-op, avoiding having to unnecessarily rebuild binaries from source.
+
+If we are in a single-configuration scenario and it built correctly, for this simple case we won't need a promotion at all,
+and just uploading directly the built packages to the ``products`` repository will be enough:
```

Comment:
I think I have a bit missing as to re-clarify how this package would then be used. As it now lives in the products remote, it won't be picked up by any job, so users that have a simpler need might not feel stuck as to what to do